### DISCUSSION TITLE
Playback Errors

### DISCUSSION THREAD

**Discussion by CranberryPie1** [itsToggle/plex_debrid#272] [General]
Hey, I just finished setting up plex debrid on my plex server and I'm getting errors related to playback, such as 

`Please check that the file exists and the necessary drive is mounted.`
`An error occurred trying to play, Error code: s1001 (Network)`

Any ideas here?

**Comment by itsToggle** [comment]
Hi, that sounds like your IP address is blocked on realdebrid. Someone on discord just expirienced this same error today - could you run the following commands from a terminal on your server and see if your IP is blocked?

`curl -4 https://real-debrid.com/vpn | grep blocked` and `curl -6 https://real-debrid.com/vpn | grep blocked`

Realdebrid does not usually allow you to access their servers from a VPS/seedbox. They block some, but not all VPS IPs. You can combat this by requesting a different IP for your server (preferably your own IPv4 address) or by running your server behind a whitelisted VPN (check out https://real-debrid.com/vpn).

**Reply by CranberryPie1** [reply]
hmm, I just ran the command on 3 of my servers hosted on different hosts and they're all blocked! I really don't want to use a VPN service, as they're pretty expensive. Do you know of any server providers that aren't blocked on Real Debrid? One of my hosts isn't known and im surprised they're blocked.

**Reply by itsToggle** [reply]
You could check out the oracle free tier VPS (which is completely free to use). Its a little tricky to setup and the resources are a bit limited (it's an ARM instance with 4x3GHz, 24GB RAM and a 4Gb/s connection), but there is a nice guide on the discord channel.

**Reply by CranberryPie1** [reply]
Damn :( Oracle is blocked

**Reply by itsToggle** [reply]
Oh that's strange, haven't heard of anyone reporting this.. could this be region specific? Could you post the output of the curl command/s?

**Reply by CranberryPie1** [reply]
The region is Montreal. I don't have Ipv6, below is Ipv4 output.

```
curl -4 https://real-debrid.com/vpn | grep blocked
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0 <div class="error-box"><strong>Your ISP or VPN provider IP address is currently blocked on our website, please contact the support</strong></div>
100 28727    0 28727    0     0  49531      0 --:--:-- --:--:-- --:--:-- 49529
```
